packages/chatbot-server-mongodb-public/src/eval/fuzzyLinkMatch.test.ts

```diff
@@ -57,4 +57,16 @@ describe("fuzzyLinkMatch", () => {
     const actual = "https://anotherdomain.com/path/";
     expect(fuzzyLinkMatch(expected, actual)).toBe(true);
   });
+
+  test("expected url has no path, but actual does", () => {
+    const expected = "https://example.com";
+    const actual = "https://example.com/path/to/resource";
+    expect(fuzzyLinkMatch(expected, actual)).toBe(false);
+  });
+
+  test("actual url has not path, but expected does", () => {
```

Comment:
```suggestion
  test("expected url has path, but actual does not", () => {
```
Slight reword so that `expected` comes first like in the other tests